### DISCUSSION TITLE
🚀 Feature Request: Disable cache for local development

### DISCUSSION THREAD

**Discussion by af-tomwilkins** [cloudflare/workers-sdk#13492] [Feature Requests]
### Describe the solution

Miniflare offers the ability to disable caching: https://miniflare.dev/storage/cache#disabling

I'd like to see this feature available through the Wrangler CLI please, as the cache can become counter-productive during local development.

Thanks!

**Comment by petebacondarwin** [comment]
I believe this is possible via the `startWorker()` API?
